src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -5,8 +5,8 @@
 
 # TODO Move this to a package shared by CLI and SDK
 from functools import total_ordering
+from importlib import import_module
 from enum import Enum
-from importlib import import_module  # pylint: disable=wrong-import-order
```

Comment:
Well, looks like the problem is solved: https://travis-ci.org/troydai/azure-cli/builds/238432295.